daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -179,6 +190,10 @@ object Node {
 
     override def requiredAuthorizers(): Set[Party] = key.maintainers
 
+    override def kind = NodeKind.LookupByKey
+    override def signatories = Set.empty
```

Comment:
Please set this to the signatories if the node finds a contract and `Set.empty` otherwise.

Reply:
successful LookupByKeys are now Fetches.